src/PosUi/alertsCounter/AlertsCounter.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { getAlertsCounter } from '../../rootReducer';
+import { resetAlerts } from './actions';
+import './AlertsCounter.scss';
+
+const AlertsCounter = ({ classname = '' }) => {
+  const counter = useSelector(getAlertsCounter);
+  const dispatch = useDispatch();
+  if (!counter) {
+    return null;
+  }
+  const value = counter > 99 ? '99+' : counter;
+  return (
+    // eslint-disable-next-line jsx-a11y/no-static-element-interactions
+    <div
+      className={`${classname} AlertsCounter`}
+      onClick={() => dispatch(resetAlerts())}
+    >
+      <div className="number">{value}</div>
+    </div>
+  );
+};
```

Comment:
What do you think about to split it to dumb component and smart component (will have a connection to redux, and make a decision about value)?

Reply:
There is no need in smarts components anymore - If needed I split the components to hooks (where I handle all logic and get things from store) and dumb component that get all data from hooks and represent it.

This component is to small and there is no need to add hooks only for 1 line